### PR TITLE
Added Structs for Bar/Drink properties

### PR DESCRIPTION
**Pull request by Raeppa**
new file: BarDrinkStructs, which contains BarStruct and DrinkStruct structs which contain the basic properties of associated classes
Changes to Bar/Drink.cs: combined a few methods for editing by utilizing structs mentioned above, as well as added a way to edit the ingredient list for a drink

### CODE REVIEW

File: Comparison Engine/Comparison Engine/Base Classes/Bar.cs [hunk 1]
```diff
@@ -20,16 +20,11 @@ public Bar(int barID, string barName, string barLocation)
...
-            barName = newName;
-        }
-
-        //Changes this bar object's location
-        public void EditBarLocation(string newLocation)
-        {
-            barLocation = newLocation;
+            if (barStruct.barName != null) this.barName = barStruct.barName;
+            if (barStruct.barLocation != null) this.barLocation = barStruct.barLocation;
         }
```
Comment (ElektrosStulpas) [inline Comparison Engine/Comparison Engine/Base Classes/Bar.cs:28]: I don't know if we should write ifs without curly brackets stylistically

File: Comparison Engine/Comparison Engine/Base Classes/Drink.cs [hunk 2]
```diff
@@ -19,10 +19,11 @@ public Drink(int drinkID, string drinkName)
...
         }
 
-        //Changes the name of this drink object
-        public void EditDrinkName(string newName)
+        //Changes properties of this Drink Object
+        public void EditDrink(DrinkStruct drinkStruct)
         {
-            drinkName = newName;
+            if (drinkStruct.drinkName != null) this.drinkName = drinkStruct.drinkName;
+            if (drinkStruct.ingredientList != null) this.ingredientList = drinkStruct.ingredientList;
```
Comment (ElektrosStulpas) [inline Comparison Engine/Comparison Engine/Base Classes/Drink.cs:26]: For now, I believe there's not much integration of ingredientList and it cannot be edited or added, but probably will do something with it in the future, just a notice

File: Comparison Engine/Comparison Engine/Base Classes/BarDrinkStructs.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    {
+        public string barName;
+
+        public string barLocation;
+
+        public BarStruct(string barName = null, string barLocation = null)
+        {
+            this.barName = barName;
+            this.barLocation = barLocation;
+        }
```
Comment (ElektrosStulpas) [inline Comparison Engine/Comparison Engine/Base Classes/BarDrinkStructs.cs:16]: why would we have default values as null?

File: Comparison Engine/Comparison Engine/Base Classes/BarDrinkStructs.cs [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    {
+        public string drinkName;
+
+        public List<string> ingredientList;
+
+        public DrinkStruct(string drinkName = null, List<string> ingredientList = null)
+        {
+            this.drinkName = drinkName;
+            this.ingredientList = ingredientList;
+        }
```
Comment (ElektrosStulpas) [inline Comparison Engine/Comparison Engine/Base Classes/BarDrinkStructs.cs:29]: Same as with bar struct, why have null default values?

**Review by ElektrosStulpas — COMMENTED** [review]
BarDrinkStruct doesn't seem fit for a file name, but we can leave it as it is for now

File: Comparison Engine/Comparison Engine/Base Classes/BarDrinkStructs.cs (hunk 3 above)
Comment (Raeppa) [reply Comparison Engine/Comparison Engine/Base Classes/BarDrinkStructs.cs:16]: Just for a simple way to know if we are changing those values, as can be seen in Bar/Drink.cs

File: Comparison Engine/Comparison Engine/Base Classes/Bar.cs (hunk 1 above)
Comment (Raeppa) [reply Comparison Engine/Comparison Engine/Base Classes/Bar.cs:28]: Fair point

File: Comparison Engine/Comparison Engine/Base Classes/BarDrinkStructs.cs (hunk 3 above)
Comment (ElektrosStulpas) [reply Comparison Engine/Comparison Engine/Base Classes/BarDrinkStructs.cs:16]: Oh, now I think I got it, completely missed it when reviewing.

File: Comparison Engine/Comparison Engine/Base Classes/Bar.cs [hunk 5]
```diff
@@ -23,8 +23,14 @@ public Bar(int barID, string barName, string barLocation)
         //Changes properties of this Bar Object
         public void EditBar(BarStruct barStruct)
         {
-            if (barStruct.barName != null) this.barName = barStruct.barName;
-            if (barStruct.barLocation != null) this.barLocation = barStruct.barLocation;
+            if (barStruct.barName != null)
+            {
+                this.barName = barStruct.barName;
+            }
+            if (barStruct.barLocation != null)
```
Comment (ElektrosStulpas) [inline Comparison Engine/Comparison Engine/Base Classes/Bar.cs:30]: Need a whitespace between ifs

**Review by ElektrosStulpas — CHANGES_REQUESTED** [review]

File: Comparison Engine/Comparison Engine/Base Classes/Drink.cs [hunk 6]
```diff
@@ -22,8 +22,14 @@ public Drink(int drinkID, string drinkName)
         //Changes properties of this Drink Object
         public void EditDrink(DrinkStruct drinkStruct)
         {
-            if (drinkStruct.drinkName != null) this.drinkName = drinkStruct.drinkName;
-            if (drinkStruct.ingredientList != null) this.ingredientList = drinkStruct.ingredientList;
+            if (drinkStruct.drinkName != null) 
+            { 
+                this.drinkName = drinkStruct.drinkName; 
+            }
+            if (drinkStruct.ingredientList != null)
```
Comment (ElektrosStulpas) [inline Comparison Engine/Comparison Engine/Base Classes/Drink.cs:29]: whitespace between these ifs as well, please

**Review by ElektrosStulpas — CHANGES_REQUESTED** [review]

File: Comparison Engine/Comparison Engine/Base Classes/Bar.cs (hunk 5 above)
Comment (Raeppa) [reply Comparison Engine/Comparison Engine/Base Classes/Bar.cs]: yes

**Pull request merged**